website/content/docs/connect/gateways/api-gateway/configuration/gatewayclassconfig.mdx

```diff
@@ -28,8 +28,8 @@ The following outline shows how to format the configurations in the `GatewayClas
     * [`grpc`](#consul-ports-grpc): integer | optional
     * [`http`](#consul-ports-http): integer | optional
   * [`scheme`](#consul-scheme): string  | optional
-* [`copyAnnotations`](#copyAnnotations): object | optional
-  * [`service`](#copyAnnotations-service): array of strings | optional
+* [`copyAnnotations`](#copyannotations): object | optional
+  * [`service`](#copyannotations-service): array of strings | optional
```

Comment:
Links targeting anything that isn't all lowercase are currently broken, so I fixed those